packages/markdown/remark/src/types.ts

```diff
@@ -39,6 +39,7 @@ export interface ShikiConfig {
 	langs?: LanguageRegistration[];
 	theme?: ThemePresets | ThemeRegistration | ThemeRegistrationRaw;
 	themes?: Record<string, ThemePresets | ThemeRegistration | ThemeRegistrationRaw>;
+	defaultColor?: string | false;
```

Comment:
Isn't the type of this `'light' | 'dark' | false`? And in that case we also need to update this in `schema.ts`.

Reply:
The type is: `'light' | 'dark' | string | false`.
I will add both light and dark literals for completeness.